docs/source/getting_started/getting_started.md

```diff
@@ -0,0 +1,226 @@
+# Overview
```

Comment:
:art: from time to time the perspective changes. in some paragraphs the narative is `you` and in some it is `we`. it's probably better to have a consistent narative and phrasing across the whole text. in some places it can also make sense to use a descriptive third person perspective instead.